nomad/structs/structs.go

```diff
@@ -895,6 +906,61 @@ func (u *UpdateStrategy) Rolling() bool {
 	return u.Stagger > 0 && u.MaxParallel > 0
 }
 
+const (
+	// PeriodicSpecCron is used for a cron spec.
+	PeriodicSpecCron = "cron"
+)
+
+// Periodic defines the interval a job should be run at.
+type PeriodicConfig struct {
+	// Enabled determines if the job should be run periodically.
+	Enabled bool
```

Comment:
Operationally it is nice to be able to have cron know about all the jobs and just enable/disable them as needed. Then you can list the cron jobs etc.